contracts/mocks/Failer.sol

```diff
@@ -19,4 +23,12 @@ contract Failer {
             array.push(i);
         }
     }
+
+    function failRequirement() public pure {
```

Comment:
Do we actually need this, considering we already have `failWithRevert()`?

Reply:
I realise that and `assert` and  `require` fail similarly. However, I thought that providing an additional test for the later could help *document* that `withMessage` is useful for both types of error-handling.

Reply:
Hm, I guess that makes sense. Alright then!